starter-code/pizza.js

```diff
@@ -1 +1,77 @@
+'use strict';
 // Write your Pizza Builder JavaScript in this file.
+
+// Produce jqeury objects (arrays) and store in var
+var $greenPeppers = $('.green-pepper');
```

Comment:
You have all your code (and therefore your variables) in the global scope. Be careful with that. Better wrap your code in a function that you will run when de window load (or the document is ready is you use jQuery).